<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sel_game</class>
 <widget class="QWidget" name="sel_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="sel_game_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 22pt &quot;Noto Sans&quot;;</string>
   </property>
   <property name="text">
    <string>sel_game_label</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="country_game">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>country_game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="region_game">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>region_game</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
